src/rockstor/system/tests/test_services.py

```diff
@@ -68,6 +68,9 @@ def test_init_service_op_valid(self):
             "nut-server",
             "rockstor-bootstrap",
             "rockstor",
+            "rockstor-replication",
+            "rockstor-scheduling",
+            "rockstor-data-collector",
```

Comment:
@Hooverdan96 We don't have these intergrated into our DB model etc. Likely we can leave them out for now. We have the others known to our backend, so adding these in here will likely throw test errors - or be inconsistent: Unless you were planning to surface these also within the Web-UI? I don't think that is a requirement in dropping supervisord however. I.e. we didn't surface it's services.

But on the other hand we may have still have some supervisord commands that will need replacing with the new equivalents you've introduced. And in which case these test entries and potential accompanying tests will then test the supervisord command replacements in-coming.

Apologies, just nipping in with a quick look and saw these. I know this is all in-draft.

Reply:
OK, just noticed you have begun work on the supervisord to systemd replacment in-code :).